spartan/aztec-network/templates/validator.yaml

```diff
@@ -50,7 +75,7 @@ spec:
           command:
             - "/bin/bash"
             - "-c"
-            - "source /shared/contracts.env && env && node --no-warnings /usr/src/yarn-project/aztec/dest/bin/index.js start --node --archiver --sequencer"
+            - "sleep 10 && source /shared/contracts.env && env && node --no-warnings /usr/src/yarn-project/aztec/dest/bin/index.js start --node --archiver --sequencer"
```

Comment:
Ideally all sleeps would be wait loops otherwise we invite mysterious flakes, but if it gets us unblocked can be iterated on

Reply:
Yeah this is to allow the k8s DNS name to get set up. It runs before the node even starts, so shouldn't invite too much flake.